DigitalOcean-public.v2.yaml

```diff
@@ -13,6 +13,11 @@ info:
 servers:
   - url: 'https://api.digitalocean.com/v2'
     description: production
+tags:
+  - name: account
+    description: Account
+  - name: ssh_keys
+    description: SSH Keys
```

Comment:
So, for example, this description can explain that it is the resource name and our thinking around resources.